front/src/applications/stdcmV2/components/StdcmInputVia.tsx

```diff
@@ -9,39 +9,44 @@ import { ISO8601Duration2sec } from 'utils/timeManipulation';
 
 import { StdcmStopTypes } from '../types';
 
+type StdcmInputViaProps = {
+  stopType?: StdcmStopTypes;
+  pathStepStopFor: PathStep['stopFor'];
+  updatePathStepStopTime: (stopTime: string) => void;
+};
+
 const StdcmInputVia = ({
   stopType,
-  pathStep,
+  pathStepStopFor,
   updatePathStepStopTime,
-}: {
-  stopType?: StdcmStopTypes;
-  pathStep: PathStep;
-  updatePathStepStopTime: (stopTime: string) => void;
-}) => {
+}: StdcmInputViaProps) => {
   const { t } = useTranslation('stdcm');
 
-  const [pathStepStopTime, setPathStepStopTime] = useState(
-    pathStep.stopFor ? `${ISO8601Duration2sec(pathStep.stopFor) / 60}` : ''
+  const computedStopTime = useMemo(
+    () => (pathStepStopFor ? `${ISO8601Duration2sec(pathStepStopFor) / 60}` : ''),
+    [pathStepStopFor]
   );
 
-  const stopWarning = stopType === StdcmStopTypes.DRIVER_SWITCH && Number(pathStepStopTime) < 3;
+  const [pathStepStopTime, setPathStepStopTime] = useState(computedStopTime);
+
+  const stopWarning = stopType === StdcmStopTypes.DRIVER_SWITCH && Number(computedStopTime) < 3;
 
   const debounceUpdatePathStepStopTime = useMemo(
-    () => debounce((value) => updatePathStepStopTime(value), 500),
+    () => debounce((value) => updatePathStepStopTime(value), 300),
     []
   );
 
   useEffect(() => {
-    let newStopTime = pathStepStopTime;
-    const isPassageTime = stopType === StdcmStopTypes.PASSAGE_TIME || stopType === undefined;
-    if (isPassageTime && pathStepStopTime !== '0') {
+    let newStopTime = computedStopTime;
```

Comment:
`pathStepStopTime` is used to keep track of user input, as the Input component doesn't handle this internally like other ui-core components do. Otherwise the pathStep will just be updated with the last caracter entered.
`stopType` is just here to determine the default value of the `computedStopTime`, we don't need to do the same for it as we don't need to update it.
I didn't want to spend a lot of time on analysing the component as it was just a small bug fix and not a refacto but i made some additional changes and it also resolved the issue mentionned above by @axrolld & @Akctarus :)